Add recipe for epx

alex-iam opened the pull request:
### Brief summary of what the package does

"epx" stands for "Emacs Project eXecutor".
Provides functionality for executing per-project commands and storing them in `dir-locals-file`. 
Intended workflow: create a command (user is prompted for name, the command itself and additional fields), and then execute it. Execution function provides completion from all created commands. Command executes either in the compilation buffer, or separate window with `shell`: that depends on `:compile` key in the command plist.

### Direct link to the package repository

https://git.sr.ht/~alex-iam/epx

### Your association with the package

I am an author and maintainer.

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.  Quick first pass -

⸺ `epx.el` with byte-compile using Emacs 30.1:
```
epx.el:131:10: Warning: the function `comint-send-string' is not known to be defined.
```

⸺ `epx.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- epx.el#L101: You may want to consider `(derived-mode-p 'xyz)`
```

⸺ Package and license:
- Avoid `:branch` in recipes except in unusual cases
- Unreachable package URL in epx.el: 'https://git.sr.ht/~alex-iam/epx'

alex-iam commented:
> ⸺ `epx.el` with byte-compile using Emacs 30.1:
> 
> ```
> epx.el:131:10: Warning: the function `comint-send-string' is not known to be defined.
> ```

Fixed ( I can only build for 30.0.92 for now, but I'm pretty sure I fixed the issue)
> 
> ⸺ `epx.el` with [melpazoid](https://github.com/riscy/melpazoid):
> 
> ```
> - epx.el#L101: You may want to consider `(derived-mode-p 'xyz)`
> ```

I feel like this doesn't have the same consequences: I want to check precisely in that case. Is it necessary to implement this change?

> 
> ⸺ Package and license:
> 
>     * Avoid `:branch` in recipes except in unusual cases

Could you please clarify then what is the default behaviour? Would it just grab the lates commit, or stick to the default branch? I added :branch to avoid building feature-branches in case that it is the first option.

> 
>     * Unreachable package URL in epx.el: 'https://git.sr.ht/~alex-iam/epx'

What do you mean? It is reachable for me. And it seems to the CI too. Could it be an issue on your side?

alex-iam commented:
:branch removed!

riscy commented:
Yes, thank you!  And welcome to MELPA. :)
